src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,238 @@
+
+        private void Dispose(bool disposing)
+        {
+            if (_disposed)
+                return;
+
+            if (disposing)
+            {
+                _infoHandle = IntPtr.Zero;
+            }
+            _disposed = true;
+        }
+
+        /// <summary>
+        /// A property to get the application ID from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <value>Application ID.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string AppId
+        {
+            get
+            {
+                string appId;
+                int ret = Interop.Stc.Info.GetAppId(_infoHandle, out appId);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get AppId from Info, Error - " + (StcError)ret);
+                    return "";
+                }
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// A property to get interface name from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <value>Interface name.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string IfaceName
+        {
+            get
+            {
+                string ifaceName;
+                int ret = Interop.Stc.Info.GetIfaceName(_infoHandle, out ifaceName);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get interface name from info, Error - " + (StcError)ret);
+                    return "";
+                }
+                return ifaceName;
+            }
+        }
+
+        /// <summary>
+        /// Get the time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <param name="from">The beginning of the time interval.</param>
+        /// <param name="to">The end of the time interval.</param>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        public void GetTimeInterval(out long from, out long to)
```

Comment:
Do not use out param

```suggestion
        public DateTime From { get; }
        public DateTime To { get; }
```

Reply:
Changed as per your suggestion.